src/diffusers/models/attention.py

```diff
@@ -187,95 +187,137 @@ def __init__(
         dim: int,
         num_attention_heads: int,
         attention_head_dim: int,
-        dropout=0.0,
+        dropout: Optional[float] = 0.0, 
         cross_attention_dim: Optional[int] = None,
         activation_fn: str = "geglu",
         num_embeds_ada_norm: Optional[int] = None,
         attention_bias: bool = False,
-        only_cross_attention: bool = False,
         upcast_attention: bool = False,
+        temb_channels: Optional[int] = None, # for ada_group_norm
```

Comment:
Not sure we need this argument :-)

Reply:
We recently added a new argument, called `norm_type` I think we can use this instead. See current main here: https://github.com/huggingface/diffusers/blob/ffb3a26c5c96faf4934adca353045c42d1da63ae/src/diffusers/models/attention.py#L201

Let's maybe try to rebase this PR :-)